### PR TITLE
unixy Callback: Use Ansible's config manager

### LINKED ISSUE #5600: plugins.callback.unixy: Warning 'CallbackModule' object has no attribute 'display_failed_stderr'
### Summary

When using the `community.general.unixy` callback plugin with a recent version of ansible I get a warning `'CallbackModule' object has no attribute 'display_failed_stderr'` and the task output is not shown.

This is a regression introduced by https://github.com/ansible/ansible/commit/dbdbfe845adac977a78c6484da1c5923754d83a2 via https://github.com/ansible/ansible/pull/77677

I already prepared a patch to fix this by using the new `self.get_option` interface and I'd be happy to open a PR with it.

### Issue Type

Bug Report

### Component Name

community.general.unixy

### Ansible Version

```console
$ ansible --version
ansible [core 2.14.0]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/kerel/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.10/site-packages/ansible
  ansible collection location = /home/kerel/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.10.8 (main, Nov  1 2022, 14:18:21) [GCC 12.2.0] (/usr/bin/python)
  jinja version = 3.1.2
  libyaml = True

```

### Community.general Version

```console (paste below)
$ ansible-galaxy collection list community.general

# /usr/lib/python3.10/site-packages/ansible_collections
Collection        Version
----------------- -------
community.general 5.8.0  
```

### Configuration

```console (paste below)
$ cat ansible.cfg
[defaults]
stdout_callback = unixy
```

### OS / Environment

Manjaro Linux 22.0

### Steps to Reproduce

Any playbook with tasks producing output will trigger this. I tested with the following:

`playbook.yml`: 
```yaml (paste below)
---
- hosts: all
  tasks:
    - ansible.builtin.ping:
```

```
$ ansible-playbook playbook.yml

# in my specific case I run inside a virtualbox using vagrant:
$ sudo vagrant provision debian_bullseye --provision-with version
```

### Expected Results

The task output should be shown.

### Actual Results

Task output is missing, but warning shown:
```console (paste below)
==> debian_bullseye: Running provisioner: version (ansible)...
    debian_bullseye: Running ansible-playbook...
Executing playbook check_version.yml

- all on hosts: all -
Gathering Facts...
[WARNING]: Failure using method (v2_runner_on_ok) in callback plugin
(<ansible_collections.community.general.plugins.callback.unixy.CallbackModule
object at 0x7fb06d861a20>): 'CallbackModule' object has no attribute
'display_failed_stderr'
ansible.builtin.ping...

- Play recap -
  debian_bullseye            : ok=2    changed=0    unreachable=0    failed=0    rescued=0    ignored=0   
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by kerel-fs**
##### SUMMARY

Start using Ansible's config manager in the unixy Callback plugin.

Fixes #5600

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

`unixy`

### CODE REVIEW

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

File: changelogs/fragments/5601-unixy-callback-use-config-manager.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - unixy callback plugin - fix plugin to work with ansible 2.14 by using Ansible's configuration manager for handling options (https://github.com/ansible-collections/community.general/issues/5600).
```
Comment (felixfontein) [inline changelogs/fragments/5601-unixy-callback-use-config-manager.yml:2]: There is no such thing as Ansible 2.14. There is ansible-core 2.14, or Ansible 7.
```suggestion
  - unixy callback plugin - fix plugin to work with ansible-core 2.14 by using Ansible's configuration manager for handling options (https://github.com/ansible-collections/community.general/issues/5600).
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@kerel-fs thanks for fixing this!

File: plugins/callback/unixy.py [hunk 2]
```diff
@@ -142,7 +142,7 @@ def v2_runner_on_ok(self, result, msg="ok", display_color=C.COLOR_OK):
             display_color = C.COLOR_CHANGED
             task_result = self._process_result_output(result, msg)
             self._display.display("  " + task_result, display_color)
-        elif self.display_ok_hosts:
+        elif self.get('display_ok_hosts'):
```
Comment (bobelev) [inline plugins/callback/unixy.py:145]: @felixfontein shouldn't self.get_option be here as well?

Comment (felixfontein) [reply plugins/callback/unixy.py:145]: Hmm, yes, looks like a typo. Do you want to create a PR?

Comment (kerel-fs) [reply plugins/callback/unixy.py:145]: @bobelev Thank you for finding this error! Yes, this is a typo I made.

I do wonder if my lack of testing did result in more errors.
I was assuming the change to be simple enough to let me do a drive-by contribution without looking into [collection_test_pr_locally](https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_test_pr_locally.html) .

just created PR #5744 to fix the typo (but again  did not have the time to test the PR locally yet).

Comment (felixfontein) [reply plugins/callback/unixy.py:145]: #5744 fixes this.

**Pull request merged**